spec/unit/rbeapi/api/alias/fixture_alias.text

```diff
@@ -0,0 +1,3 @@
+alias Alias1 my command
```

Comment:
To be honest I am a bit stucked here.

I changed the regex and I can get a multiline alias (except for this extra space).

`
irb(main):004:0> al.getall
=> {"sis"=>{:name=>"sis", :command=>" show interface status"}, "desc"=>{:name=>"desc", :command=>"\n   !! Syntax: desc <interface> <description>\n   1 conf\n   2 int %1\n   3 description %2\n   4 end"}}
`

I can create a single line alias.

`
irb(main):005:0> al.create('sid', :command=>'show interface description')
`

But if I try a multiline one:

`
al.create('desc2', :command=>'\n   !! Syntax: desc <interface> <description>\n   1 conf\n   2 int %1\n   3 description %2\n   4 end')
`

The result in the FastCli running-conf looks like this.

`
alias desc2 \n !! Syntax: desc <interface> <description>\n 1 conf\n 2 int %1\n 3 description %2\n 4 end
`

Do you have any hints for me?